index.html

```diff
@@ -4,33 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio-Setup-Mobile-V</title>
+<!-- PAGE  TITLE -->
+    <title>Martin Pineda's Portfolio</title>
     <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-    <!-- Header section -->
-    <header class="menu-line">
+<!-- BACKGROUND & NAVIGATION MENU -->
+<div class="wrapper">
```

Comment:
- Hi @AztkMarty The desktop design looks good, however, can you please place the decorative background images in their appropriate location as it is on the Figma design, Currently it overlaps the hero section test which makes it don't visible to read. 

- **Your Design**

![image](https://github.com/AztkMarty/Portfolio-Setup/assets/58771507/ec77e419-dac3-4fa4-b746-f02db9dc59c3)

-  **Fima Design**

![image](https://github.com/AztkMarty/Portfolio-Setup/assets/58771507/48918ff2-8d34-4f66-a8ad-43b691b0545b)